lib/lotus/repository.rb

```diff
@@ -674,18 +674,18 @@ def _persisted?(entity)
 
       # Add time create an entity
       def _update_created_at(entity)
-        if entity.class.attributes.include?(:created_at)
+        if entity.respond_to?(:created_at) && entity.respond_to?(:created_at=)
           entity.created_at ||= Time.now.utc
         end
       end
 
       # Add time update an entity
       def _update_updated_at(entity)
-        if entity.class.attributes.include?(:updated_at)
-          if entity.updated_at
-            entity.updated_at = Time.now.utc
+        if entity.respond_to?(:updated_at) && entity.respond_to?(:updated_at=)
+          if entity.respond_to?(:created_at) && entity.created_at
+            entity.updated_at ? entity.updated_at = Time.now.utc : entity.updated_at = entity.created_at
```

Comment:
But why here need the created_at?

do we have difference between created_at and updated_at if we use this?

`entity.updated_at = Time.now.utc`